scripts/apps/search/controllers/get-bulk-actions.tsx

```diff
@@ -26,7 +26,8 @@ export function getBulkActions(
 
     const {isLocked, isLockedByOtherUser, isPublished} = sdApi.article;
 
-    const noneLocked = articles.every((article) => !isLocked(article));
+    const noneLocked = articles.some((article) => isLocked(article)) !== true;
```

Comment:
this is actually rather `someUnlocked` than `noneLocked`

Reply:
`articles.some((article) => isLocked(article))` means `at least one is locked`.

If it is `not true`, that `at least one is locked`, then none are locked.

Reply:
I've missed that `!== true` at the end, not very readable imo

Reply:
I thought `some` would perform better than `every`, but turns out it's the same. `every` stops checking after the first negative response.